### ISSUE TITLE
Accessibility Audit for High Zoom, Colour Contrast, Screen Reader and Keyboard Navigation

### ISSUE THREAD

**Issue by m158261** [nbgallery/nbgallery#980]
In an effort to improve the overall accessibility of nbgallery I have conducted an audit in some key areas. These are keyboard navigation, high zoom levels, screen reader use, colour and contrast. I have listed some areas for improvement below. 

Any feedback and suggestions are most welcome.

**Keyboard Navigation**

 - Enter/Spacebar does not work with skip links
 - Enter/Spacebar does not work on drop down menus
 - Auto close drop down menus when user navigates away
 - Enter/Spacebar does not open notebook
 - Enter/Spacebar does not work on some buttons i.e. Admin page
 - Esc does not close drop down menus i.e. User settings menu
 - Only spacebar toggles check boxes and radio buttons

**Colour Contrast**

 - Keywords, comments and logical operators meet WCAG AA for contrast but not AAA
 - Different coloured text on forms does not meet AAA for contrast i.e. Upload notebook dialog

**High Zoom Levels (400%)**

 - Long lines in code cells wrap into line number margin

**Screen Reader**

 - No announcements for notebook recommendations 
 - Open create new group dialog announcement is confusing (announcing colours etc)
 - Does not announce the three items within Adding Search Filter tooltip
 - Magnifying glass button within search on homepage just announces 'button'

**Other Accessibility Issues**

 - Ensure unique ARIA IDs
 - Implement accessible names for buttons - linked to screen reader issues
